tools/announcement_voice_filter/README.md

```diff
@@ -0,0 +1,13 @@
+#AI Announcement Voice Effects Template#
+This template file helps apply the correct sound effects to generate further Station AI announcement messages.
```

Comment:
Make this a valid MD file. It looks bad.
![image](https://user-images.githubusercontent.com/25063394/116814414-c70a3380-ab50-11eb-8053-4e42050303ec.png)

Reply:
oof, it does look stinky. I'll fix it up

Reply:
![image](https://user-images.githubusercontent.com/12197162/116814544-6b8c7580-ab51-11eb-8b7e-a7c02b7a7cc6.png)
done